app/src/main/java/com/hayaisoftware/launcher/activities/SearchActivity.java

```diff
@@ -507,13 +507,14 @@ private void updateVisibleApps() {
     }
 
     private void sortApps() {
+        Collections.sort(mActivityInfos, mAlphabeticalOrderComparator);
+
         if (mShouldOrderByRecents) {
             Collections.sort(mActivityInfos, mRecentOrderComparator);
         } else if(mShouldOrderByUsages) {
             Collections.sort(mActivityInfos, mUsageOrderComparator);
-        } else {
```

Comment:
I only removed the trailing else and put the alphabetical sorting above the if. Should work as expected now.